src/main/java/org/opentripplanner/routing/graph/Graph.java

```diff
@@ -945,19 +962,19 @@ public void summarizeBuilderAnnotations() {
         }
     }
 
-    public GraphMetadata getMetadata() {
+    public GraphEnvelope getMetadata() {
         // Lazy-initialize the graph metadata since it is not serialized.
```

Comment:
The comment still says "metadata" which is not wrong since the envelope is metadata, but still should probably be changed to say "envelope".